glean/tests/utils.ts

```diff
@@ -153,3 +153,20 @@ export class CounterUploader extends Uploader {
     };
   }
 }
+
+export class TimeoutTaskMockUploader extends Uploader {
+
+  public count = 0;
+
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  async post(_url: string, _body: string): Promise<UploadResult> {
+    this.count++;
+    return new Promise<UploadResult>(reslove => {
+      setTimeout(() => {
+        reslove(new UploadResult(UploadResultStatus.RecoverableFailure));
+      }, 1500);
+    });
+
+  }
+
+}
```

Comment:
Small things
- Typo in `resolve`
- If you prefix a variable name with `_` you don't also need the eslint-ignore, they accomplish the same thing
- Formatting

```suggestion

export class TimeoutTaskMockUploader extends Uploader {
  public count = 0;

  async post(_url: string, _body: string): Promise<UploadResult> {
    this.count++;
    return new Promise<UploadResult>((resolve) => {
      setTimeout(() => {
        resolve(new UploadResult(UploadResultStatus.RecoverableFailure));
      }, 1500);
    });
  }
}

```